.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Release
+
+on:
+  push:
+    branches: master
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Use Node.js v14
+      uses: actions/setup-node@v1
+      with:
+        node-version: 14.x
+    - name: Install dependencies
+      run: npm ci
+    - name: Test
+      run: npm run test --if-present
+    - name: Build
+      run: npm run build --if-present
+    - name: Release
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+      run: npx semantic-release
```

Comment:
I'm assuming this command does-it-all, i.e.
- makes & commits git tag (based on package.json version)
- makes github release with commit log since last release
- publishes NPM package

Reply:
Yep!